pyVmomi/SoapAdapter.py

```diff
@@ -1076,23 +1076,12 @@ def __init__(self, proxyPath):
    # @param cert_file The SSL certificate file to use when wrapping the socket.
    # @param kwargs In case caller passed in extra parameters not handled by
    #        SSLTunnelConnection
-   def __call__(self, path, key_file=None, cert_file=None, **kwargs):
+   def __call__(self, path, *args, **kwargs):
       # Only pass in the named arguments that HTTPConnection constructor
       # understands
-      tmpKwargs = {}
-      for key in http_client.HTTPConnection.__init__.__code__.co_varnames:
-         if key in kwargs and key != 'self':
-            tmpKwargs[key] = kwargs[key]
-      tunnel = http_client.HTTPConnection(path, **tmpKwargs)
-      tunnel.request('CONNECT', self.proxyPath)
-      resp = tunnel.getresponse()
-      if resp.status != 200:
-        raise http_client.HTTPException("{0} {1}".format(resp.status, resp.reason))
-      retval = http_client.HTTPSConnection(path)
-      retval.sock = _SocketWrapper(tunnel.sock,
-                                   keyfile=key_file, certfile=cert_file)
-      return retval
-
+      conn = http_client.HTTPSConnection(path, context=ssl._create_unverified_context())
```

Comment:
Hi @goneri , Thanks for the PR.
Why unverified context by default? 
Please can you brief about this change?

Reply:
Oh, my bad. I forgot to check that. It should be better now.